docs/references/architecture/adr-112-proposer-based-timestamps.md

```diff
@@ -193,14 +194,22 @@ type SynchronyParams struct {
 
 #### Compatibility parameters
 
-In order to ensure backwards compatibility, PBTS should be enabled using a consensus parameter.
+In order to ensure backwards compatibility, PBTS should be enabled using a consensus parameter:
 
 ```go
-type XXXParams struct {
-        EnableHeight    int64 `json:"enable_height"`
+type FeatureParams struct {
+        PbtsEnableHeight int64 `json:"pbts_enable_height"`
+        ...
 }
 ```
 
+The semantics is identical to the one adopted to enable vote extensions via
```

Comment:
```suggestion
The semantics is similar to the one adopted to enable vote extensions via
```

Reply:
Vote extensions are more complex (H is special, different from H+1, which is not the case with PBTS)

Reply:
Addressed.